lib/ledger_sync/adaptors/quickbooks_online/adaptor.rb

```diff
@@ -195,55 +202,24 @@ def self.new_from_oauth_client_uri(oauth_client:, uri:, **override)
           )
         end
 
+        def self.resource_from_ledger_type(type:)
+          API_RESOURCE_MAPPING_OVERRIDE_REVERSE[type.downcase] || LedgerSync.resources[type.downcase.to_sym]
+        end
+
         private
 
         def oauth_base_uri
           @oauth_base_uri ||= "#{root_uri}/v3/company/#{realm_id}"
         end
 
-        def oauth(force: false)
-          if @oauth.nil? || force
-            @oauth = OAuth2::AccessToken.new(
-              oauth_client.client,
-              access_token,
-              refresh_token: refresh_token
-            )
-          end
-
-          @oauth
-        end
-
-        def oauth_client
```

Comment:
Will deleting this method break above references? Such as `authorization_url` on line 53

```
def authorization_url(redirect_uri:)
  oauth_client.authorization_url(redirect_uri: redirect_uri)
end
```

Reply:
Good catch.  I moved them back for now.  I don't love that they live here, but I'd rather tackle that later.